.github/workflows/build_and_test.yml

```diff
@@ -258,12 +258,13 @@ jobs:
       # Run tests on all Postgres versions in release builds and only on the latest version in debug builds
       # run without LFC on v17 release only
       test-cfg: |
```

Comment:
Need to adjust the comment above as well

Reply:
Good catch, thanks